DB/GADPerson.m

```diff
@@ -95,6 +101,50 @@ -(GADPerson *)initWithDictionary: (NSDictionary *) dict {
     return people;
 }
 
++ (void) fetchPersonInfoWithCriteria:(NSDictionary*)crit andUsername:(NSString*)usrname andPassword:(NSString*) pw completionHandler:(void(^_Nonnull)(NSArray<GADPerson *> *))completion {
+
+    NSMutableArray * queryItems = [NSMutableArray array];
+    for (NSString * key in crit){
+        [queryItems addObject:[NSURLQueryItem queryItemWithName:key value:crit[key]]];
+    }
+    NSURLComponents *components = [NSURLComponents componentsWithString:@"https://itwebappstest.grinnell.edu/DotNet/WebServices/api/db"];
```

Comment:
Can we declare this as a constant at the top of the file? `static const NSString *apiURL = @"...";` and then `NSURLComponents *components = [NSURLComponenets componentsWithString:apiURL];` here.